view commit

yosukehirai90 opened the pull request:
https://gyazo.com/c9ad5665de26168ee465e777d01e6ffb

yosukehirai90 committed:
view commit

TOMARUMARU commented on app/assets/stylesheets/application.css.scss:
```diff
@@ -0,0 +1,29 @@
+@import "modules/main-form";
+@import "modules/main-chat";
+
+html, body, .wrapper, .side-bar, .main {
```
application.scssはimportするだけにして、他の記述は別ファイルに切り分けましょう。

hiroki5123 replied:
こちらまだ修正されてません。

TOMARUMARU commented on app/views/layouts/mailer.text.haml:
```diff
@@ -0,0 +1 @@
+= yield
```
mailerファイルなど使用しないものは削除しましょう。

TOMARUMARU reviewed (commented):
まずコミットをきちんと分けましょう。その際コミット名もきちんと何をしたか命名しましょう。

PR名もしっかり命名しましょう。何をこのブランチで実装したのか見る人がわかるようにです。

また、WHATとWHYもきちんと書きましょう！

ファイルの最後は改行を入れましょう。数ファイルで入ってないです！

yosukehirai90 committed:
view commit

ビューに関するファイルを編集しました。

hiroki5123 commented on app/views/messages/_message.html.haml:
```diff
@@ -0,0 +1,5 @@
+  %h3.message__name seo
+  %p.message__date 2019/02/01 05:23:12
+  %p.message__body Goodbye World
+  
```
最後に空行いれてください。

hiroki5123 commented on app/views/messages/_sidebar.html.haml:
```diff
@@ -0,0 +1,16 @@
+
+      %h3.group__group-name sampleグループ
+      %p.group__group-log まだメッセージがありません
+      
```
最後に空行いれてください。

hiroki5123 commented on app/views/messages/index.html.haml:
```diff
@@ -0,0 +1,29 @@
+  %head
+    %meta{ charset: "utf-8" }
+    = stylesheet_link_tag "application"
+  %body
```
ここより上はapplication.html.hamlに記載されていると思いますが、重複して書いているのはなぜですか？

hiroki5123 commented on app/views/messages/index.html.haml:
```diff
@@ -0,0 +1,29 @@
+              = fa_icon("image")
+            = file_field_tag :image, class: "input-box__file-send" 
+          = submit_tag "Send", class: "form__send-btn"
+          
```
最後に空行いれてください。

ghost commented:
WHATとWHYもきちんと書きましょう！
最初なんの実装をしたのかわかりませんでした。
指摘されたコメントに対しては必ず返信をしてください。

yosukehirai90 committed:
view commit

#WHY
ビュー画面を作るため

#WHAT
指摘されたところを修正
index.html.hamlとapplication.html.hamlに記載されているところは重複していません。どちらも必要で、ない場合エラーが出ます。

yuki-uchida reviewed (approved):
LGTMですが、頂いたコメントには必ずリプライする形で行った対応を記述してください
